andela/casablanca-rc
# #157311127 Consumer to consumer funding

#### What does this PR do?
allow users transfer funds from their wallets to other people's wallets
#### Description of Task to be completed?
* create a server side method that deducts the amount to be transferred from the initiator's wallet and adds it to the recipient's wallet
* call the method in the client side passing in the values the user entered as arguments
#### How should this be manually tested?
* clone the repo and change into the project directory
* start the application(instructions in the readme)
* visit the application in the browser
* login/signup as a user
* visit your profile
* there should be a wallet section that displays your current balance and also an option to fund the wallet
* fund your wallet
* create a new user
* go to your profile
* in the `Wallet` section, enter the email of the other user you created and an amount to be transferred
* click `Send`
* you should get a toast message notifying you of the result of your transfer request
using the admin access, enable wallet payment in the payment dashboard.
#### Any background context you want to provide?
n/a
#### What are the relevant pivotal tracker stories?(if applicable)
#157311127
